src/core/opamConsole.ml

```diff
@@ -442,6 +442,12 @@ let carriage_delete =
   else
     carriage_delete_unix
 
+let url ~ref ~label =
+  if not (color ()) || Sys.win32 && get_win32_console_shim `stdout Mode = Shim then
```

Comment:
Might be worth a comment linking to the handy https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda which describes the escape codes for Linux